packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json

```diff
@@ -1,6 +1,6 @@
 {
   "general": {
-    "credentialName": "Legal Entity",
+    "credentialName": "Verified Legal Entity",
```

Comment:
Same as identity flow

Reply:
this is the name for the credential, issue with double verified etc is fixed by removing that in the localization strings